### ISSUE TITLE
Fresh install 1.4.1 on macOS 10.15.4 produces AppleEvent timed out error

### ISSUE THREAD

**Issue by humanatwork** [JuliaLang/julia#35878]
Hi Team! Excited to get started here, but ran into the following issue on Mac during install. Attempting to open Julia.app to bring up the REPL produces the following:

![image](https://user-images.githubusercontent.com/13501438/81870074-2a1b4880-9543-11ea-8a1e-e54f44e4ca5c.png)

Specs:
macOS: 10.15.4
Julia version: 1.4.1

**UPDATE: RESOLVED**
Leaving this post here in case anyone runs into the same issue. Not a permanent solution, but if you run `exec /Applications/Julia-1.4.app/Contents/Resources/julia/bin/julia` or simply navigate to that location within the Julia-1.4.app package contents and run the executable, it resolves. I suspect it's an initialization permission issue with Terminal, as after performing that explicit execution the Julia app alias now runs directly from Applications.

**Comment by ViralBShah** [comment]
Are you using a non-standard shell or have something that is not what we expect as a default? Is this reproducible somehow - like if you reinstall Julia, does it still timeout?

**Comment by jakobnissen** [comment]
Just experienced this exact issue with a fresh 1.5.0 install on MacOS. Issue was resolved by re-downloading and re-installing Julia.

**Comment by why7682** [comment]
Encountered the same issue in 1.5.0. The problem was fixed after I choose "Show Package Contents" and then went to /Applications/Julia-1.5.app/Contents/Resources/julia/bin and click the executable file "julia". Then when I closed, the Julia-1.5 application can run. Here's another method that may help
change the code in /Applications/Julia-1.5.app/Contents/Resources/Scripts/main.scpt:
`set RootPath to POSIX path of (path to me)
tell application id "com.apple.terminal"
	with timeout of 300 seconds
		do script ("exec '" & RootPath & "Contents/Resources/julia/bin/julia'")
		activate
	end timeout
end tell`

**Comment by alexbakus** [comment]
Had the same issue on 1.7.1 mac intel 64bit binary. Using the solution from @why7682 with `Show Package Contents` and navigating to `~/Contents/Resources/julia/bin` and running the julia package resolved for me as well.

**Comment by kerrykey** [comment]
I just had this issue with Julia 1.7.2 on an Intel Mac running macOS 12.3.1. I tried opening Julia three times and got the same time-out message each time. I started to try the solution method in OP but when I clicked on (the not running) terminal app, three Julia REPLs immediately opened, i.e., one per previous time-out attempt. Exited terminal app, re-clicked Julia app and now the REPL launches without any issues.
